experimental/ActionPolyRing/src/Content.jl

```diff
@@ -227,6 +229,8 @@ factor(apr::ActionPolyRingElem) = __wrap_factorization_apr(factor(data(apr)), pa
 
 base_ring(apr::ActionPolyRing) = base_ring(__upr(apr))
```

Comment:
In fact I'd argue that "right" definition here would be
```suggestion
base_ring(apr::ActionPolyRing) = __upr(apr)
```
Or rather: `__upr` should be replaced by `base_ring`. The "base ring" or a ring `R` is a ring from which `R` was constructed.